requirements.txt

```diff
@@ -1,6 +1,6 @@
-aiohttp >= 3.9.3, < 4
+aiohttp >= 3.12.15, < 4
 python-dateutil >= 2.9.0, < 3
-setuptools >= 69.1.1
+setuptools >= 78.1.1
 build >= 1.2.1, < 2
-urllib3 >= 1.26.19, < 3
+urllib3 >= 1.26.19, != 2.0.*, != 2.1.*, != 2.2.0, != 2.2.1, < 3
```

Comment:
I know this looks rather nuts, but it is, in fact, the right approach for targeting >= 1.26.19 and >= 2.2.2 specifically, to address dependency vulnerabilities